create/create.js

```diff
@@ -0,0 +1,39 @@
+import { checkAuth, createClient, getWorkshops, logout } from '../fetch-utils.js';
+
+checkAuth();
+
+const form = document.querySelector('.client-form');
+
+form.addEventListener('submit', async e => {
+    e.preventDefault();
+    const formData = new FormData(form);
+    const dawi = { name: formData.get('client-name'), workshopId: formData.get('workshop-id') };
+    //console.log(dawi);
+    await createClient(dawi);
+    
+    form.reset();
+});
```

Comment:
Here we can add location.replace('../workshops') to return the user to the workshop page. Keep in mind that the folder name and the file.js name should match in order for this to work. Excellent job Brenden!